dotcom-rendering/src/client/ophan/ophan.ts

```diff
@@ -134,7 +157,7 @@ export const abTestPayload = (tests: ServerSideTests): OphanABPayload => {
 	return { abTestRegister: records };
 };
 
-export const recordPerformance = (): void => {
+export const recordPerformance = async (): Promise<void> => {
```

Comment:
Would we want to keep this a synchronous function and pass `record` as a dependency?